modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -296,11 +296,11 @@ void RenderableOrbitalKepler::render(const RenderData& data, RendererTasks&) {
     //glDepthMask(false);
```

Comment:
Just above this, the two lines declaring `nrOrbits` and `vertices` are no longer needed